myapp/lib/passport-config.js

```diff
@@ -8,19 +8,15 @@ passport.serializeUser((email, done) => {
   done(null, email);
 });
 
-passport.deserializeUser((email, done) => {
-  connection.query(
-    'SELECT * FROM users WHERE email=?',
-    [email],
-    (error, results) => {
-      if (results.length > 0) {
-        // ユーザーを正しく取得できた
-        done(null, results[0]);
-      } else {
-        done(null, error);
-      }
-    }
-  );
+passport.deserializeUser(async (email, done) => {
+  let user;
+  try {
+    user = await User.findOne({ where: { email: email } });
```

Comment:
12行目のlet定義削除し、この行でconst定義することが可能です。
なるべくletを使用しない設計を心がけると良いです。
```suggestion
  try {
    const user = await User.findOne({ where: { email: email } });
```